src/resources/account/Preferences.js

```diff
@@ -0,0 +1,36 @@
+import Resource from '../Resource'
+
+export default class Preferences extends Resource {
+  constructor (api) {
+    super(api)
+    this.endpoint = 'user/preferences'
+  }
+
+  async list () {
+    return await this.request({
+      method: 'GET',
+      path: this.endpoint
+    })
+  }
+
+  async retrieve (key) {
+    return await this.request({
+      method: 'GET',
+      path: `${this.endpoint}/${key}`
+    })
+  }
+
+  async create (params = {}) {
```

Comment:
Why is it called `create` ? Create will imply that you're creating preferences. But it can also mean that you're updating an existing 'preference'. Why not just call it `update` or `set` ?